src/global_distrib/mod_global_distrib_hosts_refresher.erl

```diff
@@ -31,6 +31,13 @@
 %% gen_mod API
 -export([deps/2, start/2, stop/1]).
 
+%% Test & debug API
+-export([pause/0, unpause/0]).
+
+-record(state, {
+          refresh_interval :: pos_integer(),
+          tref :: reference() | undefined
```

Comment:
Since it's a timer reference, can't we call it `timer_ref`? Not much longer and much more descriptive in my opinion.

Reply:
It's pretty common in MIM source code to abbreviate it as `tref`.